docs/BR.md

```diff
@@ -391,6 +396,8 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 **P-Label**: A XN-Label that contains valid output of the Punycode algorithm (as defined in RFC 3492, Section 6.3) from the fifth and subsequent positions.
 
+**Primary Network Perspective**: The Network Perspective used by the CA to make the determination of 1) the CA's authority to issue a Certificate for the requested domain(s) or IP address(es) and 2) the Applicant's authority and/or domain authorization or control of the requested domain(s) or IP address(es). 
```

Comment:
I wonder what should be considered the primary perspective, if there are any limitations at all. 

It seems intuitive that the primary perspective is the "main" or "existing" network from which the determinations are being made. However, from a continuity perspective I can imagine there are circumstances where one of the network perspectives could be promoted to the primary, so maybe we want to add flexibility?

Reply:
I always thought of the primary perspective as the perspective that is audited and compliant with netsec requirements.

I think flexibility in allowing other perspectives to be promoted to primary (presuming these perspectives are audited and meet all existing netsec requirements) would be good. @chrisbn do you feel the current draft does not allow this flexibility and is there any language you would recommend adding to make this explicit?

Reply:
I think conceptually any of the network perspectives should be able to act as the primary perspective. It depends on the interpretation of "audited and meeting netsec requirements", if those the items in the "SHOULD" section, I think there's proper flexibility and the interpretation I would support. If the intent is to expect a full netsec audit for the primary node, that's another interpretation.

Reply:
In general I support flexibility in the selection of perspectives and designation of a primary perspective. However, I think it is very important to maintain this distinction between the primary and the additional perspectives, and require a netsec audit on the primary perspective. To significantly reduce the implementation burden, this draft does not require additional perspectives to be audited and comply with netsec (beyond the baseline security expectations contained in this section). If done improperly, this could undermine the security protections established by the netsec requirements as an adversary could target these additional perspectives and use them to fraudulently validate domain control.

To avoid this, the primary perspective MUST be netsec audited and properly complete domain control validation. In today's BRs, this is sufficient domain control validation to sign a certificate. From there, multi-perspective checking can block a signature in the event of a BGP attack, but the additional, non-audited perspectives have no way of overriding a domain-control-validation request that has not been successfully completed by the primary perspective. This ensures that none of the procedures added in this document degrade the existing security of the PKI.

Thus, I feel my interpretation would be that the "primary perspective" must have a full netsec audit as this is the perspective that must succeed for issuance. If a CA has multiple fully audited perspectives, I think it is fine to use them on the same request and the primary perspective can be chosen from the audited perspectives (although preferably before the request is sent and not after the CA observes the domain control validation results).

Reply:
Marking this thread as resolved, as discussion carries into https://github.com/ryancdickson/staging/pull/8#discussion_r1370359911. Comment was addressed in https://github.com/ryancdickson/staging/pull/8#discussion_r1372196235 and a minor clarifying update was described in https://github.com/ryancdickson/staging/pull/8#discussion_r1374523955.